packages/rest/src/body-parsers/body-parser.ts

```diff
@@ -32,7 +34,7 @@ export class RequestBodyParser {
   readonly parsers: BodyParser[];
 
   constructor(
-    @inject.tag(REQUEST_BODY_PARSER_TAG, {optional: true})
+    @inject(filterByTag(REQUEST_BODY_PARSER_TAG), {optional: true})
```

Comment:
What is the reasoning for this change? Is it just a stylistic refactoring, or does it introduce any changes in the externally-observed behavior, e.g. the order in which the parsers are invoked?

I am ok with the change, but would like to better understand the consequences.

Reply:
It's stylistic. IIRC, we would like to deprecate `@inject.tag` in the future and use `@inject` with a filter function instead.